backend/testfiles/execution/stdlib/json.dark

```diff
@@ -299,8 +403,8 @@ Json.parse<Uuid> "\"00000000-0000-0000-0000-00000000000\"" = PACKAGE.Darklang.St
 
 
 // Bytes
-//Json.serialize<bytes> Bytes.empty = PACKAGE.Darklang.Stdlib.Result.Result.Ok "\"\""
-//Json.parse<bytes> "\"\"" = PACKAGE.Darklang.Stdlib.Result.Result.Ok Bytes.empty
+Json.serialize<Bytes> Bytes.empty = PACKAGE.Darklang.Stdlib.Result.Result.Ok "\"\""
+Json.parse<Bytes> "\"\"" = PACKAGE.Darklang.Stdlib.Result.Result.Ok Bytes.empty
 
 // Passwords - TODO
```

Comment:
Couldn't find good tests for parsing/serializing passwords

Reply:
I guess passwords shouldn't be convertible to json, so test that it gets an Error?